### PR TITLE
runtime(deb822sources): Add minimal ftplugin

### PR DESCRIPTION
**Pull request by jamessan**
Set comment related options and avoid automatic line wrapping.

### CODE REVIEW

**Review by dkearns — APPROVED** [review]

**Comment by chrisbra** [comment]
thanks!

File: runtime/ftplugin/deb822sources.vim [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+" Last Change:  2024 Mar 20
+" License:      Vim License
+" URL:          https://salsa.debian.org/vim-team/vim-debian/blob/main/ftplugin/deb822sources.vim
+
+if exists('b:did_ftplugin')
+  finish
+endif
+let b:did_ftplugin=1
+
+setlocal comments=:#
```
Comment (chrisbra) [inline runtime/ftplugin/deb822sources.vim:12]: Wondering if you rather would like to use `comments=b:#` instead?

File: runtime/ftplugin/deb822sources.vim [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+setlocal commentstring=#%s
```
Comment (chrisbra) [inline runtime/ftplugin/deb822sources.vim:13]: Same here? commentstring=#\ %s

**Review by chrisbra — APPROVED** [review]

**Comment by chrisbra** [comment]
I had 2 tiny comments, but I am fine with it and I'll merge it now. I guess you can consider this for later (or not at all :) )

**Pull request merged**